### PR TITLE
Add Hello

### PR DESCRIPTION
**Pull request by SevaGostev**
I've added 'Hello' to y.txt

### CODE REVIEW

File: y.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-V1
+v99
```
Comment (mikegostev) [inline y.txt:1]: WTF? Why do you need it?

**Review by mikegostev — CHANGES_REQUESTED** [review]
Fix it please

File: y.txt (hunk 1 above)
Comment (SevaGostev) [reply y.txt:1]: Sorry, typo

**Review by mikegostev — APPROVED** [review]
That's ok now

**Pull request merged**